ndcube/ndcube.py

```diff
@@ -886,28 +886,32 @@ def _new_instance_from_op(self, new_data, new_unit, new_uncertainty):
     def __array_ufunc__(self, ufunc, method, *inputs, **kwargs):
         if method == '__call__':
             if ufunc == np.add:
-                if isinstance(inputs[0], NDCube):
+                if isinstance(inputs[0], NDCube) and kwargs.get("dunder"):
```

Comment:
Is `"dunder"` a standard kwarg or custom for this implementation?  And what does it mean?

Reply:
I have used the `dunder` (not a standard kwarg) kwarg here to distinguish between the operators and numpy ufuncs to properly handle operator overloading.